edge/pkg/edgehub/clients/factory.go

```diff
@@ -39,11 +45,10 @@ func GetClient(clientType string, config *config.EdgeHubConfig) Adapter {
 			ProjectID:        config.CtrConfig.ProjectID,
 			NodeID:           config.CtrConfig.NodeID,
 		}
-		return quicclient.NewQuicClient(&quicConfig)
-	} else {
-		log.LOGGER.Errorf("donot support client type: %s", clientType)
+		return quicclient.NewQuicClient(&quicConfig), nil
+	default:
+		log.LOGGER.Errorf("Do not support client type: %s", clientType)
```

Comment:
What about the error message,:

"Client type: %s is not supported"

Reply:
+1